### PR TITLE
Signup login homepage week8

### PR DESCRIPTION
**Pull request by simaoalicechen**
1. Added two tests to check if Login and Signup components exist, not failing; 
2. Added post and get tests without api from previous format, failing without data, but succeeding if using shortlist api;
3. Added posts and get tests with axios using api format, failing might be due to not complete backend data; 

What to do next: 

1. I can't add more easy tests on other components, such as buttons, for now, since my local codes are different from last demo's codes. I will use another PR request to add ids to the team repo codes so that we can test on those components.

### CODE REVIEW

**Comment by waltage** [comment]
coverage folder is obscuring the PR; not sure why it's where it's at and not handled by the .gitignore like the rest of our runs, but it needs to be removed

File: src/frontend/shortlist/src/components/layout/signup_login/Login.vue [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<script>
+import NavbarLogin from "./NavbarLogin.vue";
+import Logo from "./Logo.vue";
+
```
Comment (waltage) [inline src/frontend/shortlist/src/components/layout/signup_login/Login.vue:4]: This is missing the pinia state manager that Pooja and Adelaide added... this whole file is quite a bit different?

https://github.com/gcivil-nyu-org/team-2-inperson/blob/develop/src/frontend/shortlist/src/components/layout/signup_login/Login.vue

**Review by waltage — CHANGES_REQUESTED** [review]
This whole PR looks like it is a remerge of #120

File: src/frontend/shortlist/src/components/layout/signup_login/Login.vue (hunk 1 above)
Comment (simaoalicechen) [reply src/frontend/shortlist/src/components/layout/signup_login/Login.vue:4]: > This is missing the pinia state manager that Pooja and Adelaide added... this whole file is quite a bit different?
> 
> https://github.com/gcivil-nyu-org/team-2-inperson/blob/develop/src/frontend/shortlist/src/components/layout/signup_login/Login.vue

Ye told me pinia is a temporary solution and therefore we don’t have to worry about it.

**Comment by simaoalicechen** [comment]
> This whole PR looks like it is a remerge of #120

That’s weird. I might have pushed a wrong button or something. Currently, I don’t have a computer. I will clear this up in about an hour.

**Comment by simaoalicechen** [comment]
I will delete this PR once the coverage and command issue is solved.

**Pull request closed without merging**